docs: add mermaid diagrams redo

CBID2 opened the pull request:
## Linked Issue
Closes #516 
<!--

If you have a pull request related to a current issue please link to that issue number.

That issue can be linked to the pull request by using the side panel in the Github UI or using the `#` symbol followed by the number of the associated issue.

To link a pull request to an issue to show that a fix is in progress and to automatically close the issue when someone merges the pull request, type the keyword "Closes" followed by a reference to the issue. For example, Closes #404 or Closes Virtual-Coffee/virtualcoffee.io/issues/404.

-->

## Description

This is a redo of pull request #520. I created Mermaid diagrams to create visual workflows of three major community processes:

-  #### Lightning Talk Project Workflow — docs/lightning-talks/processes/project-workflow.md

<img width="2940" height="8820" alt="image of new diagram" src="https://github.com/user-attachments/assets/a79fca53-d9a6-4cd8-b693-7219f1415049" />

-  #### Facilitating a Challenge — docs/monthly-challenges/facilitate-a-challenge.md

<img width="2940" height="10903" alt="image of new diagram" src="https://github.com/user-attachments/assets/ae837feb-7bb0-46ef-90b0-62c48f349526" />

- #### Lunch & Learn Guidelines — docs/lunch-and-learns/process-docs.md


<img width="2940" height="12480" alt="image of new diagram" src="https://github.com/user-attachments/assets/ca686aee-7ffa-4e77-887d-0aa2ed924e52" />



These diagrams provide an at-a-glance overview of how each process flows, making it easier for new volunteers and contributors to understand their roles and responsibilities.

@BekahHW, I would love your feedback on this. 

<!--

A pull request description describes what constitutes the Pull Request and what changes you have made to the code.

It explains what you've done, including any code changes, configuration changes, migrations included, new APIs introduced, changes made to old APIs, any new workers/crons introduced in the system, copy changes, and so on. You get the gist.

A good description informs everyone that is reaading it of the purpose of the pull request. This helps not just the current maintainers but anyone reading it now or in the future to understand your intent.

If the request is not complete but you want feedback use  Draft Pull Request option of the Pull request dropdown menu.

@mention individuals that you want to review the PR, and mention why. (“ @username I want to know what you think of this code.”)

-->

## Methodology
Following the official [Docusaurus v3 Mermaid integration guide](https://docusaurus.io/docs/markdown-features/diagrams), I
:

1. Added `@docusaurus/theme-mermaid` to the `docusaurus.js.config` file and enabled Markdown support.
2. Translated three existing textual workflows into clear, concise Mermaid diagrams.
3. Ensured consistency in visual style and logical flow across all diagrams.
4.  Tested each Markdown file to confirm correct rendering in local preview.
 
These changes improve readability, onboarding efficiency, and process transparency by combining structured documentation with visual context.

<!--

This section explains why the above changes explained were done.

Sometimes a developer feels that it's okay to write "Business/Product requirement" in the description. That's fine, but doing so defeats the purpose of this section.

If there is a better explanation as to why the changes were suggested, it's always good to attach a document reference link for that information.

A good "Why" section should explain the reasoning behind any changes.

-->

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

CBID2 committed:
docs: add mermaid diagrams part 2

Signed-off-by: Christine <shecoder30@gmail.com>

CBID2 committed:
Prettified Code!

CBID2 committed:
docs: redid format

Signed-off-by: Christine <shecoder30@gmail.com>

CBID2 committed:
Prettified Code!

CBID2 committed:
docs: fixing deployment issues

Signed-off-by: Christine Belzie <105683440+CBID2@users.noreply.github.com>

CBID2 committed:
Prettified Code!

BekahHW commented:
@CBID2 I like the direction this is heading! It looks like the only file update with the diagram is the Lunch and Learns. You might need to commit and push the latest changes. 

To make it easier to read and remove a lot of the white space, can we update the Mermaid diagram to use a two-column horizontal flow with subgraphs?

This version groups related steps into logical sections (Planning, CFP, Prep, Event, Post-Event) and displays them left to right rather than top to bottom. It makes the diagram more compact and scannable.

Here’s an _**example**_ of the structure to follow:

```
flowchart LR
  %% Planning
  subgraph A[🛠 Planning]
    A1[📢 Call for Team] --> A2[👥 Onboard Team]
    A2 --> A3[📝 Assign Coordinators]
  end

  %% CFP
  subgraph B[📬 CFP]
    B1[📣 CFP Opens] --> B2[🧾 Collect Proposals]
    B2 --> B3[👩‍💻 Grouping]
  end

  %% Speaker Prep
  subgraph C[🧭 Prep]
    C1[🗂 Organize Talks] --> C2[✅ Checklists]
    C2 --> C3[📸 Speaker Info]
    C3 --> C4[📁 Shared Folder]
  end

  %% Event
  subgraph D[🎤 Event]
    D1[🧾 Final Prep] --> D2[📨 Directions]
    D2 --> D3[🧑‍🏫 Talks]
  end

  %% Post
  subgraph E[🎥 Post-Event]
    E1[☁️ Upload to YouTube]
  end

  %% Flow
  A3 --> B1
  B3 --> C1
  C4 --> D1
  D3 --> E1
```


Also, when I'm looking at the L&L file, the text is getting cut off: 
<img width="536" height="435" alt="image" src="https://github.com/user-attachments/assets/1d143147-01fb-4d75-890c-e9ad9ea3cbf6" />

I'm on chrome. Not sure which browser you're using, but that's something to investigate.

CBID2 committed:
Merge branch 'Virtual-Coffee:main' into adding-diagrams-redo

CBID2 committed:
docs: add other diagrams

CBID2 committed:
Prettified Code!

CBID2 committed:
docs: change and resize diagrams

CBID2 commented:
> @CBID2 I like the direction this is heading! It looks like the only file update with the diagram is the Lunch and Learns. You might need to commit and push the latest changes.
> 
> To make it easier to read and remove a lot of the white space, can we update the Mermaid diagram to use a two-column horizontal flow with subgraphs?
> 
> This version groups related steps into logical sections (Planning, CFP, Prep, Event, Post-Event) and displays them left to right rather than top to bottom. It makes the diagram more compact and scannable.
> 
> Here’s an _**example**_ of the structure to follow:
> 
> ```
> flowchart LR
>   %% Planning
>   subgraph A[🛠 Planning]
>     A1[📢 Call for Team] --> A2[👥 Onboard Team]
>     A2 --> A3[📝 Assign Coordinators]
>   end
> 
>   %% CFP
>   subgraph B[📬 CFP]
>     B1[📣 CFP Opens] --> B2[🧾 Collect Proposals]
>     B2 --> B3[👩‍💻 Grouping]
>   end
> 
>   %% Speaker Prep
>   subgraph C[🧭 Prep]
>     C1[🗂 Organize Talks] --> C2[✅ Checklists]
>     C2 --> C3[📸 Speaker Info]
>     C3 --> C4[📁 Shared Folder]
>   end
> 
>   %% Event
>   subgraph D[🎤 Event]
>     D1[🧾 Final Prep] --> D2[📨 Directions]
>     D2 --> D3[🧑‍🏫 Talks]
>   end
> 
>   %% Post
>   subgraph E[🎥 Post-Event]
>     E1[☁️ Upload to YouTube]
>   end
> 
>   %% Flow
>   A3 --> B1
>   B3 --> C1
>   C4 --> D1
>   D3 --> E1
> ```
> 
> Also, when I'm looking at the L&L file, the text is getting cut off: <img alt="image" width="536" height="435" src="https://private-user-images.githubusercontent.com/34313413/500534835-1d143147-01fb-4d75-890c-e9ad9ea3cbf6.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjAzOTg4MTYsIm5iZiI6MTc2MDM5ODUxNiwicGF0aCI6Ii8zNDMxMzQxMy81MDA1MzQ4MzUtMWQxNDMxNDctMDFmYi00ZDc1LTg5MGMtZTlhZDllYTNjYmY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDEzVDIzMzUxNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjNGNjMDI1NDljM2EwMzBjMDlmN2E0M2NmNDg1MDYwZjgzZjk2YmFkZjlkNjg2MzI1YWZjOGNhZWQwZmEzZGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.LJxOt0ujsjBvCfbUIkS4CIJtO-k-a-98Ax5BqAasEyY">
> 
> I'm on chrome. Not sure which browser you're using, but that's something to investigate.

Hi @BekahHW! I made the changes you requested. The only issue I have now is how to make the merimad diagrams bigger so users don't have to use the Zoom in feature on the browser. Can you help with that?

BekahHW commented:
@CBID2 can you tell me what you've tried or link to the resources you looked up so far?

CBID2 committed:
Merge branch 'Virtual-Coffee:main' into adding-diagrams-redo

CBID2 committed:
docs: unify Mermaid diagrams and styling
- Global: default 20px Mermaid, spacing 50/70
- CSS: mermaid SVG responsive width
- Lunch & Learns: compact vertical flowchart + minimal loop; tighter size
- Monthly Challenge: concise labels; add Proofread OK? and More updates? loops
- Lightning Talks: loops for CFP/Speaker Prep; reduced padding/spacing
- Copy: clarify intros/captions

Signed-off-by: Christine Belzie <shecoder30@gmail.com>

CBID2 committed:
Prettified Code!

CBID2 commented:
> @CBID2 can you tell me what you've tried or link to the resources you looked up so far?

Hi @BekahHW! :) I made the diagrams bigger

CBID2 commented:
Hey @adiati98! Can you review this? 😁

adiati98 reviewed (changes requested):
Hey @CBID2 ,

I'm currently on vacation, so I can't take a look at this closer.

But I've seen some diagrams on my phone:

1. Some arrow parts are behind the box as screenshot below:

   ![Screenshot_20251023_210201_DuckDuckGo.jpg](https://github.com/user-attachments/assets/e09702f8-55e1-4486-af84-ec8e6fec1bb2)

2. Some boxes in a diagram are way too big, while in the following diagram, the boxes are small. Probably make them a bit proporsional? On the phone, it looks like some diagrams are on the left side and not centered. 

   ![Screenshot_20251023_210343_DuckDuckGo.jpg](https://github.com/user-attachments/assets/df77d57f-d113-4840-9ed1-b7cc60403048)

3. We never do automation for monthly challenges. Where did you read this on the docs?

   ![Screenshot_20251023_210505_DuckDuckGo.jpg](https://github.com/user-attachments/assets/9fdf6759-f753-4807-a641-095ad55d4bce)

Can you please read and confirm with the docs when creating the diagram? If you need help, let us know. 🙂

---

@BekahHW, can you take a closer look at this? TIA! ✨️

BekahHW commented:
These are looking really good! A couple of concerns:
- Because they're vertical, they take up a lot of space and so the users might miss the actual documentation. It's worth exploring a horizontal path. 
- I'm a bit worried that the diagrams aren't clear and also want to make sure they do align with the processes documented. The key here is that it should make it easier for the user to grok the process. If they look at the diagrams and can't really understand what to expect, then we've missed the mark. Ask yourself if you could generally coordinate the events based on the diagrams you've created.

CBID2 committed:
style: update Mermaid diagrams and section styling for consistency

- Standardized section headings and spacing
- Improved Mermaid diagram readability
- Made CFP section more prominent
- Ensured consistent sizing across all sections

CBID2 committed:
Prettified Code!

CBID2 committed:
docs: add enhanced Mermaid diagrams to documentation

- Added detailed Mermaid flowcharts to project workflow, monthly challenges, and lunch & learns
- Improved visual representation of processes and workflows
- Enhanced documentation with clear visual aids for better understanding

CBID2 committed:
Prettified Code!

CBID2 commented:
> These are looking really good! A couple of concerns:
> 
> * Because they're vertical, they take up a lot of space and so the users might miss the actual documentation. It's worth exploring a horizontal path.
> * I'm a bit worried that the diagrams aren't clear and also want to make sure they do align with the processes documented. The key here is that it should make it easier for the user to grok the process. If they look at the diagrams and can't really understand what to expect, then we've missed the mark. Ask yourself if you could generally coordinate the events based on the diagrams you've created.

Hey @BekahHW. I think I got this time

adiati98 commented:
I feel like it'd be more useful if the diagram is inside its section than standalone at the beginning of the page. It can be before the written docs or after for a better visual. The reason is that it'll be easier for folks to see and read both the docs and diagram in the same place rather than to scroll back and forth between the diagram and written docs.

@BekahHW any thoughts on this?

dominicduffin1 commented:
@CBID2 I have some comments on the Monthly Challenge docs. I realize existing documentation is not fully representative of what we are doing but since we are making updates I'd like to ensure we are reflecting current processes.

---

I'm generally concerned that the diagrams suggest a level of process organization that we are not really doing right now. In an ideal world we would be doing it or something equivalent but with limited bandwidth on the team we often fall back on simpler processes.

---

### Announcements

I have a template I use for the Slack announcement instead of going through a draft -> feedback -> approval process every month.

I'm not aware of any social media posts announcing monthly challenges.

I'd like to get input from the @Virtual-Coffee/maintainers on the "Send to Email List" step as I don't have visibility into this but I'm not sure it is correct to imply a direct link between the Announcement tasks and the email list as I don't do anything in this regard as part of my work on the announcement.

### Post challenge

The diagram makes a specific link between "Community Engagement" and "Gather Feedback" but we don't currently have any process to gather feedback from the community. I'd like to have such a process but we don't have the bandwidth to do this effectively at present and I'd prefer the docs to represent reality rather than aspiration.

We do update documentation to reflect what we did during the challenge where new elements were introduced that year. I think the "Update Documentation" should be directly linked from the "During Challenge" section as this is not something that occurs as a result of community engagement or feedback.

We do from time-to-time gather feedback from within the team post-challenge. This feeds into planning of future challenges. Ideally the diagrammatic structure would reflect that flow between team feedback and planning in a circular flow leading back to the earlier stages.

### Team Cooridnation

I'm concerned that this workflow is stating the obvious without really giving any information about what we actually do. Having said that, since how we do this isn't really documented, this is not your fault. We can probably revisit this section in future.

---

We also update the `#monthly-challenge` Slack channel topic and description each month. This step is missing from the diagrams. Maybe we could have a "Day 1" diagram that includes posting the announcement and updating the Slack channel as I usually do these two tasks at the same time.

I'd also like to see a link between the website updates and the announcement as I lift one or two sentences from the website to fill into my announcement template and add to the channel description.

---
> Let us know your availability for the challenge in the reminder thread that drops every mid-month in the #vc-monthly-challenge-team channel in Slack.

We don't use this reminder thread any more, although it is still posted by Slackbot each month.  Ideally, since we are making updates I'd like to remove references to this, although it can be done separately.

---

@adiati98 @BekahHW Please feel free to chime in if you disagree with anything I've said or feel it doesn't accurately reflect your roles in our processes.

adiati98 commented:
@dominicduffin1, I think as long as the diagram based on the workflow that are stated in the docs, that should be fine for now. That means, we need to update our docs to follow the current workflow in another issue.

But I do agree with you for some flows that **aren't based on the docs** here:

1. **Announcements:**

   - We don't announce monthly challenges on social media. 
   - We never do e-mail list to send out monthly challenges in any form. --> @Virtual-Coffee/maintainers can you please confirm?
   - We don't need get feedback & revise, also final approval.

2. **Post Challenge:**

   - As stated in the docs: Provide feedback in the `#vc-monthly-challenge-team` channel. --> This channel is our team's private channel. That said, we gather feedback from our team members, and not the community members.

3. **Team Coordination:**

   It seems like it's meant for the flow of [Coordinating with the Coffee Table Groups](https://deploy-preview-521--vc-community-docs.netlify.app/docs/monthly-challenges/facilitators-docs/facilitate-a-challenge). However, the diagrams don't capture the flow as per docs.

---

@BekahHW can you also please check the diagrams for L&L and Lightning Talk?

---

@CBID2 I'd like to know how did you come up with the workflow? Same as Bekah, my concern is the diagrams are not clear and not align with the workflows documented here. Please let us know if you have any questions.

dominicduffin1 commented:
@adiati98 @CBID2 Agreed that we can focus on representing the docs as currently written for this issue.

A couple more points i still think should be addressed:

1. I don't think there should be a link between feedback and documentation in the post challenge flow. Our current docs correctly don't imply this link.
2. We should include updating the challenge channel along with announcements as this is included in the written docs

CBID2 commented:
Hey @adiati98 and @dominicduffin1, I took your advice and revised my diagram for the Facilitating a Challenge docs here: https://mermaidviewer.com/share/rK_eOP-lHtMT0ib8YZzog
What do you think?

dominicduffin1 commented:
@CBID2 I think this is getting closer. Overall, I think this version of the flow represents things fairly accurately. 

I still have some feedback:

1. A minor one about word choice: We refer to those who lead teams as "Lead" not "Leader". There was a wider conversation about this a while back but I believe we concluded in favor of that. Otherwise, I think the planning flow is fine.
2. We need to put back the step of updating the website in the promotion flow. Also, I think we should say that the challenge is announced in #announcements and have a separate step for updating the #monthly-challenge channel.
3. Another minor one in the check-ins section: Mondays at 10am is goal setting not progress updates.

@adiati98 @BekahHW Do you have any other thoughts?

BekahHW commented:
I added the `hacktoberfest-accepted` label to make sure you get credit for this, @CBID2. I'm thinking that we might want to put the diagrams in a [details section](https://docusaurus.io/docs/markdown-features#details), so people can opt into looking at it. I think this makes sense, especially if someone is on a mobile device. These get a bit lengthy.

CBID2 committed:
Update Lunch & Learn process docs and flowchart

- Updated the request review section with new roles
- Enhanced the scheduling workflow with conditional paths
- Improved flowchart readability with better formatting
- Fixed Mermaid.js syntax for better rendering

CBID2 committed:
Prettified Code!

CBID2 committed:
Update Lightning Talks workflow with improved Mermaid diagrams and mobile-friendly styling

CBID2 committed:
Prettified Code!

CBID2 committed:
Update Mermaid diagrams for better readability and mobile-friendliness

CBID2 committed:
Prettified Code!

CBID2 commented:
> I added the `hacktoberfest-accepted` label to make sure you get credit for this, @CBID2. I'm thinking that we might want to put the diagrams in a [details section](https://docusaurus.io/docs/markdown-features#details), so people can opt into looking at it. I think this makes sense, especially if someone is on a mobile device. These get a bit lengthy.

Hey @BekahHW, @dominicduffin1, and @adiati98, I finally made the changes: 

### Lighting Talks 
<img width="2940" height="8385" alt="image of diagram in Lighting Talks" src="https://github.com/user-attachments/assets/095a896e-282e-4d16-8166-2fa157883646" />

### Lunch and Learn

<img width="2940" height="1664" alt="image of diagram in Lunch and Learn" src="https://github.com/user-attachments/assets/cb51d6f5-5b4c-4594-94a7-7952cd5dcb2e" />


### Facilitate a Challenge

<img width="2940" height="1664" alt="image of diagram in Facilitate a challenge" src="https://github.com/user-attachments/assets/b4d3e9e3-7804-408e-b230-e1374f365bd6" />

BekahHW commented on docs/lightning-talks/processes/project-workflow.md:
```diff
@@ -35,13 +35,226 @@ tags:
 
 The function of the Lightning Talks Event is to allow opportunities for members of the community to have an intimate space to present on a tech topic for 5-15 minutes. When live-streaming as a members-only event, speakers can feel more comfortable as it's _their_ community listening and watching. However, their is still opportunity to impact a larger audience once the videos have been processed a ...
 
-### Specifics
```
I like the way these look and how you can open and close. I would put them under the proper heading. For instance, you have all the drop downs at the top, but the CFP diagram should go in the CFP section. That helps to keep the info organized in a logical way. Otherwise the right nav for the page gets super confusing. You have the same thing 2x and you have something that says Post-Event and then CFP comes after that. 

If some of those don't have their own section rn, just move them to where they'd make sense chronologically. But really like how these have turned out!

<img width="926" height="167" alt="Image" src="https://github.com/user-attachments/assets/6d32ed8a-41dc-4372-8f9d-cd73da48e593" />

<img width="974" height="262" alt="Image" src="https://github.com/user-attachments/assets/298dbf30-a90d-41e0-b9a5-b9237565e098" />

<img width="293" height="402" alt="Image" src="https://github.com/user-attachments/assets/451870bd-8f36-45bd-a511-14f8d8a46fb1" />

CBID2 replied:
Made the changes. What do you think?

<img width="2940" height="8564" alt="image of changes to my pull request" src="https://github.com/user-attachments/assets/07627db6-8df9-4ca9-baf3-0b4d7dd97437" />

BekahHW commented on docs/lightning-talks/processes/project-workflow.md:
```diff
@@ -74,7 +287,7 @@ The function of the Lightning Talks Event is to allow opportunities for members
   (For ref: Bc there is a lag, if you’re watching and up next, you’ll be introduced before the next talk is over.)
 
-### Communication
+### Communication {#communication-section}
```
I appreciate you making these clickable!

BekahHW commented on docs/monthly-challenges/facilitators-docs/facilitate-a-challenge.md:
```diff
@@ -37,6 +37,80 @@ Let us know your availability for the challenge in the reminder thread that drop
 
 If you have ideas for a new challenge(s), you can propose and discuss them in the `#vc-monthly-challenge-team` channel on Slack or open a new [discussion on GitHub](https://github.com/Virtual-Coffee/VC-Community-Docs/discussions).
 
+Below are diagrams of the Monthly Challenge workflow. Click to expand each section to view the detailed workflow.
```
Looks great!

BekahHW commented on package.json:
```diff
@@ -17,17 +17,18 @@
     "prism-react-renderer": "^2.3.0",
-    "react": "^19.0.0",
-    "react-dom": "^19.0.0"
+    "react": "^18.3.1",
```
What's going on here?

CBID2 replied:
Hey @BekahHW. When I was working on the PR, the mermaid diagram feature was not working initially due to compatibility issues, so I had to downgrade React from 19 to 18.3.1. I also changed the order of the diagrams like you asked. [Here they are](https://github.com/Virtual-Coffee/VC-Community-Docs/pull/521#discussion_r3004163353)

BekahHW replied:
I'm not super comfortable downgrading the version. Can you explore some other solutions and present them?

CBID2 replied:
Hi @BekahHW,

Thanks for the feedback! I did some exploring and it turns out the problem isn't really about version compatibility at all. The Docusaurus Mermaid theme is looking for an optional dependency called `@mermaid-js/layout-elk` that's not installed, and that's what's causing the build to fail.

As far as solutions, here's what I came up with:

**Option 1: Add the missing dependency**
```bash
pnpm add @mermaid-js/layout-elk
```
This would be the simplest fix - just installing what the theme is expecting to find.

**Option 2: Upgrade to the latest Docusaurus version**
```bash
pnpm add @docusaurus/core@latest @docusaurus/preset-classic@latest @docusaurus/theme-mermaid@latest
```
Sometimes, newer versions have better dependency handling.

**Option 3: Convert to static images**
We could export the diagrams as PNG/SVG files and just include them as regular images in the docs.

**Option 4: Disable the ELK layout feature**
There's a config option we could add to turn off the specific feature that's causing the issue.

Personally, I think Option 1 would be suited to solve your concern. It's the most straightforward and keeps everything working as intended. What do you think?

BekahHW replied:
1. Preferred: React 19 + Docusaurus Mermaid + simpler Mermaid diagrams.
2. Acceptable if required: React 19 + Docusaurus Mermaid + @mermaid-js/layout-elk.

CBID2 replied:
Ok @BekahHW, I went with option 1, just like you preferred.

CBID2 committed:
fix: reorganize mermaid diagrams into proper sections

- Move Planning Phase diagram to Timeline > Call for coordinators section
- Move CFP diagram to existing CFP section
- Move Speaker Prep diagram to Organizing talks subsection
- Move Event diagram to Week of subsection
- Add Post-Event diagram after Communication section

Addresses feedback to place diagrams under their proper headings
instead of grouping them at the top, improving navigation and
logical organization of the documentation.

CBID2 committed:
fix: remove Post-Event section and diagram

- Remove Post-Event section that wasn't part of original process
- Keep only the core process sections: Timeline, CFP, Organizing talks, Week of, Communication
- Maintains proper diagram placement within relevant sections

devin-ai-integration[bot] committed:
Restore React 19 and simplify Mermaid diagrams

- Restore react/react-dom to ^19 (per maintainer feedback)
- Pin webpack to 5.102.1 to avoid webpackbar/ProgressPlugin incompatibility in newer webpack
- Simplify Mermaid diagrams: rely on site-wide mermaid config for styling, remove per-diagram init/classDef overrides, inline step chains, dedupe redundant edges
- Trim redundant/conflicting Mermaid CSS and remove stray code fence

Co-Authored-By: Christine Belzie <shecoder30@gmail.com>

devin-ai-integration[bot] committed:
Prettified Code!
